### PR TITLE
CR-1081642 After fuzzing shell and running validate, platform gets st…

### PR DESCRIPTION
**Pull request by xdavidz**
…uck in errored state

Based on the comments from the CR, we should explicitly re-set the u64 size for kdma cmd.

### CODE REVIEW

**Review by houlz0507 — APPROVED** [review]
I assume the size will never beyond 32-bit, correct?

File: src/runtime_src/core/include/ert.h [hunk 1]
```diff
@@ -180,7 +180,7 @@ struct ert_start_copybo_cmd {
   uint32_t state:4;          /* [3-0], must be ERT_CMD_STATE_NEW */
   uint32_t unused:6;         /* [9-4] */
   uint32_t extra_cu_masks:2; /* [11-10], = 3 */
-  uint32_t count:11;         /* [22-12], = 15, exclude 'arg' */
+  uint32_t count:12;         /* [22-12], = 16, exclude 'arg' */
```
Comment (stsoe) [inline src/runtime_src/core/include/ert.h:183]: huh?  4+6+2+12+5+4 = 33  
count has to match ert_packet, I am not following this change?

**Review by stsoe — CHANGES_REQUESTED** [review]

**Comment by xdavidz** [comment]
> I assume the size will never beyond 32-bit, correct?

This is a quick fix that existing code will only pass in 32-bit size. If we change the existing interface to 64bit, there will be too many other code changes. I just want to minimize the code change scope for this time before the 2020.2 release.

In the future, we should consider switching the interface to 64-bit.

File: src/runtime_src/core/include/ert.h (hunk 1 above)
Comment (xdavidz) [reply src/runtime_src/core/include/ert.h]: sorry, reverted back.

**Review by stsoe — APPROVED** [review]

**Comment by xdavidz** [comment]
retest this please

**Review by mamin506 — APPROVED** [review]

**Pull request merged**